app/src/main/java/de/tum/in/tumcampusapp/component/other/generic/activity/ProgressActivity.kt

```diff
@@ -238,6 +319,11 @@ abstract class ProgressActivity(
         onRefresh()
     }
 
+    override fun onStop() {
+        super.onStop()
+        apiCall?.cancel()
```

Comment:
why `onStop()`?

Reply:
You’re right, this didn’t make sense. Moved it to `onDestroy()`.